service/collector/nodepool.go

```diff
@@ -13,13 +13,33 @@ import (
 
 var (
 	nodePools *prometheus.Desc = prometheus.NewDesc(
-		prometheus.BuildFQName(namespace, subsystemCluster, "nodepools"),
+		prometheus.BuildFQName(namespace, subsystemNodePool, "count"),
```

Comment:
I think this makes more sense?

Reply:
...or is this useless metric and could be constructed by counting `desired_workers` grouped by `node_pool_id`?

Reply:
The metrics naming makes sense to me. Having metrics for the current and desired state makes also sense to me.

Reply:
I think removing this metric might be a bit too micro-optimization compared to readability that alerts based on this would yield from direct value instead of indirect computation. I think I'll keep this.